tools/nitrogfx/gfx.c

```diff
@@ -719,9 +719,11 @@ void ReadNtrPalette(char *path, struct Palette *palette, int bitdepth, int palIn
     bitdepth = bitdepth ? bitdepth : palette->bitDepth;
 
     size_t paletteSize = (paletteHeader[0x10]) | (paletteHeader[0x11] << 8) | (paletteHeader[0x12] << 16) | (paletteHeader[0x13] << 24);
-    if (palIndex == 0) {
+    if (palIndex == 0)
```

Comment:
Revert to 1TBS

Reply:
1tbs?

Reply:
have to disagree with this, as nitrogfx is mainly in Allman style (although somewhat inconsistently), until clang-format is merged (still on my todo list) and a linter is properly setup I am going to label this as out of scope, once those two are done I will ensure that nitrogfx complies for the future
(basically there are a few critical bug fixes here and a new feature among the renamings, and I want to get these in asap as the bug prevents the entire repo from building under certain systems such as cygwin)

Reply:
main.c line 175, cast to unsigned char

Reply:
there also is main.c line 291 correction of width to palindex